packages/terra-date-picker/src/react-datepicker/index.jsx

```diff
@@ -480,15 +480,15 @@ class DatePicker extends React.Component {
     if (this.props.withPortal) { event.preventDefault() }
   }
 
-  handleChange = (event) => {
+  handleChange = (event, dateValue) => {
     if (this.props.onChangeRaw) {
-      this.props.onChangeRaw(event)
+      this.props.onChangeRaw(event, dateValue)
       if (event.isDefaultPrevented()) {
         return
       }
     }
     this.setState({ inputValue: event.target.value })
```

Comment:
Does `event.target.value` here also need to be changed to `dateValue`? Same as in line 492.

Reply:
Changed! [0b496eb](https://github.com/cerner/terra-framework/pull/1004/commits/0b496eb729d1b0fc5851cc50d820b24a267a67ec)